.github/workflows/build-package.yml

```diff
@@ -28,14 +28,13 @@ jobs:
           - ubuntu/bionic
           - ubuntu/focal
           - ubuntu/jammy
-          - ubuntu/kinetic
```

Comment:
What about Ubuntu 24.04 (Noble Numbat) ?

Reply:
We haven't added support for it